python/src/trezorlib/btc.py

```diff
@@ -279,6 +279,16 @@ def copy_tx_meta(tx: messages.TransactionType) -> messages.TransactionType:
 
             res = client.call(messages.TxAck(tx=msg))
 
+        elif res.request_type == R.TXORIGINPUT:
```

Comment:
this might be folded into `if res.request_type in (R.TXINPUT, R.TXORIGINPUT)`, right?

Reply:
Done https://github.com/trezor/trezor-firmware/pull/1292/commits/ecbe988141d8806b1ce8746063c65a7ff25bd57c